common/Resources/ti.internal/extensions/ti/ti.ui.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * Appcelerator Titanium Mobile
+ * Copyright (c) 2019 by Axway, Inc. All Rights Reserved.
+ * Licensed under the terms of the Apache Public License
+ * Please see the LICENSE included with this distribution for details.
+ */
+
+let colorset;
+let osVersion;
+
+// As Android passes a new instance of Ti.UI to every JS file we can't just
+// Ti.UI within this file, we must call kroll.binding to get the Titanium
+// namespace that is passed in with require and that deal with the .UI
+// namespace that is on that directly.
+let uiModule = Ti.UI;
+if (Ti.Android) {
+	uiModule = kroll.binding('Titanium').Titanium.UI;
+}
+
+uiModule.SEMANTIC_COLOR_TYPE_LIGHT = 'light';
+uiModule.SEMANTIC_COLOR_TYPE_DARK = 'dark';
+uiModule.semanticColorType = Ti.UI.SEMANTIC_COLOR_TYPE_LIGHT;
+
+uiModule.fetchSemanticColor = function fetchSemanticColor (colorName) {
+	if (!osVersion) {
+		osVersion = parseInt(Ti.Platform.version.split('.')[0]);
+	}
+
+	if (Ti.App.iOS && osVersion >= 11) {
+		return Ti.UI.iOS.fetchSemanticColor(colorName);
+	} else {
+		if (!colorset) {
+			try {
+				colorset = require('/semantic.colors.json'); // eslint-disable-line import/no-absolute-path
```

Comment:
If we load the JSON file like we do in the link below, then we won't run into a `require()` error when running via Android Studio, Xcode, or Visual Studio. This way the native dev team here can debug via our IDEs without issue.
https://github.com/appcelerator/titanium_mobile/blob/master/common/Resources/ti.internal/bootstrap.loader.js#L31

Reply:
iOS works fine after 08a592fa7ae81e9db39fc0a68c4f25e0cd33c4c0, I'm also able to build from Android Studio just fine after updating the `ti.main.js` file in TitaniumTest to one from this PR. I believe as long as rollup is configured correctly it shouldn't be an issue